git-glossary.md

```diff
@@ -23,17 +28,20 @@ place where the full version historyof the software is stored (like CVS or Subve
 of the code is also a repository that can contain the full history of all changes.
 
 ## github
-`github` is a website and a service that maintains git repositories for project teams. Developers can interact with repositories
-atored in `gtihub` using it's web interface or using the `git` command line interface.
+`github` is a website and a service that maintains remote git repositories for project teams. Developers can interact with 
+these remote repositories stored in `gtihub` using the web interface. If you clone a repository to your development
```

Comment:
A few minor suggestions: "stored in `gtihub` using the web interface" -> "stored on github using their web interface"

Reply:
Agreed. Made the change.